vic/drivers/image/include/vic_driver_image.h

```diff
@@ -187,6 +203,7 @@ void vic_start(void);
 void vic_store(void);
 void vic_write(void);
 char will_it_snow(double *t, double t_offset, double max_snow_temp,
-                  double *prcp, size_t n);
+                  double *prcp,
+                  size_t n);
```

Comment:
0.59!
I will update my version to 0.60 and try it again. I will let you know tomorrow. Still weird :)